base/challenge-skeleton/challenge/Dockerfile

```diff
@@ -21,4 +21,4 @@ RUN ln -s /secrets/flag /chroot/flag
 
 COPY files /chroot/home/user
 
-CMD kctf_setup && drop_privs nsjail --config /config/nsjail.cfg
+CMD kctf_setup && drop_privs nsjail --config /kctf/nsjail.cfg --config /config/nsjail.cfg
```

Comment:
we probably should call them base.nsj and chal.nsj ? wdyt?

just something to clarify that one is the base and the other is on top.. since order matters

Reply:
I don't think we should invent a new file extension so I would still go with .cfg as in the nsjail docs.
Maybe /kctf/nsjail_base.cfg and /config/nsjail.cfg?

Reply:
Yea that works :-)

Reply:
done